src/admin/usageStatuses/usageStatuses.component.tsx

```diff
@@ -49,24 +51,35 @@ function UsageStatuses() {
         header: 'Value',
         accessorFn: (row) => row.value,
         id: 'value',
+        filterVariant: 'text',
+        filterFn: 'fuzzy',
+        columnFilterModeOptions: [
+          'fuzzy',
+          'contains',
+          'startsWith',
+          'endsWith',
+          'notEquals',
+          'betweenInclusive',
```

Comment:
```suggestion
```
Doesn't make sense for text value

Reply:
It provides some useful filtering, albeit limited for text values.

If a user wanted to filter for Catalogue Items (Cameras 10 to Cameras 20) precisely, they could with `betweenInclusive`. `fuzzy `and `startsWith `would include other Cameras as well in the results.

You are less likely to find a use case like that for Usage Status values, however I included it anyway since I thought all text columns having the same filter functions throughout would make sense. I do recognise that the use for `betweenInclusive `is limited for text values, so I don't mind removing it from the list.

Reply:
The mock data used for unit tests is not reflective of data the actually users would put in. So can you remove and don't add to any of the other text value columns. The only text value column this would be applicable for is "serial number" in the items table